x-pack/plugins/features/common/kibana_feature.ts

```diff
@@ -142,6 +142,13 @@ export interface KibanaFeatureConfig {
     description: string;
     privileges: readonly ReservedKibanaPrivilege[];
   };
+
+  /**
+   * Indicates whether the feature is available as a standalone feature. The feature can still be
+   * referenced by other features, but it will not be displayed in any feature management UIs. By default, all features
+   * are enabled.
+   */
+  disabled?: boolean;
```

Comment:
__note:__ I'm not a fan of `disabled`-name (although I don't have a better suggestion), but we already had `disabled` for privileges so I'm just trying to maintain consistency.

Reply:
Yeah, as I mentioned, it was mostly because of the existing `disabled` at the feature privilege and sub-feature privilege levels, but `hidden` for a feature sounds good to me as long as it's fine by you. @legrego, any objections to renaming `feature.disabled` to `feature.hidden`? I'm fine with either option at this point. Since `disabled` is only for Serverless and only for config, we can change it without consequences anytime if we want to.

Reply:
@azasypkin I have no objection to renaming `feature.disabled` to `feature.hidden`